packages/@dcl/inspector/src/components/EntityInspector/EntityInspector.tsx

```diff
@@ -37,12 +41,19 @@ export const EntityInspector = withSdk(({ sdk }) => {
   }
   useChange(handleUpdate, [entity])
 
-  const inspectors = [SceneInspector, TransformInspector, GltfInspector]
+  const inspectors = [
+    { name: EditorComponentNames.Scene, component: SceneInspector },
+    { name: 'core::Transform', component: TransformInspector },
```

Comment:
maybe we can use the componentName instead of the literal string here in case of a typo or adding a new component with diff name types ? (I remembered this happens to us with the Selector component).
`sdk.components[componentName].componentName`